src/main/java/org/apache/sling/resourcemerger/impl/picker/ResourceTypeHierarchyBasedResourcePicker.java

```diff
@@ -99,6 +100,11 @@ public List<Resource> pickResources(ResourceResolver resolver, String relativePa
         return resources;
     }
 
+    @Activate
+    protected void activate(final Configuration config) {
+
```

Comment:
Maybe add a comment why this is necessary, e.g. reference https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#service.component-ordering.generated.properties and state that otherwise the component properties are not populated from the ObjectType defaults.

Reply:
Done